### ISSUE TITLE
Deactivating high contrast mode does not respect originally selected theme

### ISSUE THREAD

**Issue by takouhai** [dangeredwolf/ModernDeck#346]
Hey, hey! Just installed this using the [`moderndeck-bin` AUR package](https://aur.archlinux.org/packages/moderndeck-bin), but I was also able to reproduce this on the Flatpak. I'm running Arch Linux (btw). This feels like a styling thing, so it probably affects more platforms than mine...

After setting a theme...
![20220908_211733](https://user-images.githubusercontent.com/814940/189252870-b1b01fb9-5172-4646-80ef-5460657a19e2.png)
...and switching to High Contrast mode...
![20220908_211756](https://user-images.githubusercontent.com/814940/189253041-e23d9163-ba1f-48d1-b73a-f79567a31e84.png)
![20220908_211807](https://user-images.githubusercontent.com/814940/189253040-77a466fd-34b6-454a-86b8-3c504cd4ef0e.png)
...when you switch BACK, ModernDeck doesn't respect the theme you originally selected and goes back to the default (but with the right highlight color).
![20220908_211816](https://user-images.githubusercontent.com/814940/189253091-9d2355e1-eb45-43a7-99a0-3b0cc2eb4da6.png)
![20220908_211828](https://user-images.githubusercontent.com/814940/189253090-890e97dc-3782-4078-ab4e-8bcff3625f1f.png)

Let me know if you need more info.

Loving the app, by the way!

**Comment by dangeredwolf** [comment]
Yeah this behavior is kind of for technical reasons as high contrast uses AMOLED as a base. I should probably figure out a way to handle this better for users?
